### PR TITLE
Fix: diskless - Update readme.rst regarding GPG keys

### PR DESCRIPTION
**Pull request by oxedions**
Fix issue #384

### CODE REVIEW

File: roles/core/diskless/readme.rst [hunk 1]
```diff
@@ -242,25 +244,25 @@ Entering the livenet module will prompt the following menu:
 
 In this menu you can do 3 actions:
 
-* Generate a new nfs staging image : A staging image is the base image. You must not boot onto a stagging image but firsty create a golden image from it and boot on the golden image specific filesystem (Created with option 3).
-* Generate a new nfs golden image from a staging image : Create a golden image from previoulsy created staging image.
-* Manage nodes of a golden image: Create a specific file system for each nodes for a specific golden image. After adding a node to a golden image via this option, you can boot the node onto the golden image.
+* Generate a new nfs staging image : A staging image is the base image. You must not boot onto a stagging image but firstly create a golden image from it and boot on the golden image specific filesystem (Created with option 3).
```
Comment (hmeAtos) [inline roles/core/diskless/readme.rst:247]:
```suggestion
* Generate a new NFS staging image : A staging image is the base image. You must not boot into a staging image but firstly create a golden image from it and boot on the golden image specific filesystem (Created with option 3).
```

**Review by hmeAtos — CHANGES_REQUESTED** [review]

File: roles/core/diskless/readme.rst [hunk 2]
```diff
@@ -242,25 +244,25 @@ Entering the livenet module will prompt the following menu:
...
+* Generate a new nfs golden image from a staging image : Create a golden image from previously created staging image.
```
Comment (hmeAtos) [inline roles/core/diskless/readme.rst:248]:
```suggestion
* Generate a new NFS golden image from a staging image : Create a golden image from previously created staging image.
```

**Review by hmeAtos — CHANGES_REQUESTED** [review]

**Comment by oxedions** [comment]
Thanks a lot @hmeAtos !! 😊

**Review by johnnykeats — APPROVED** [review]

**Review by ginomcevoy — APPROVED** [review]
Looks good to me!

**Comment by oxedions** [comment]
Thanks guys !
Merging.

**Pull request merged**